src/client/types.d.ts

```diff
@@ -83,7 +83,7 @@ export interface PviBucket {
 }
 
 export interface EvaluateMetricWithValue extends BaseEvaluateMetric {
-  readonly type: "fraction" | "percent" | "count" | "pvi";
+  readonly type: "fraction" | "percent" | "count" | "pvi" | "pvibydistrict";
```

Comment:
With the changes in this PR, the `"pvi"` type is no longer used:
```suggestion
  readonly type: "fraction" | "percent" | "count" | "pvibydistrict";
```